### ISSUE TITLE
how to make it really non-intrusive?

### ISSUE THREAD

**Issue by pps83** [sakra/cotire#106]
Non-intrusive is listed as the first feature in the list. But `cotire(MyExecutable)` makes it totally intrusive IMO :) I don't want to cotire everything, and I don't want that "non-intrusive" step to decide what should be in precompiled header. I already have precompiled header and everything works fine, how can I enable that precompiled header for CMake project with cotire? Morever, I have a few projects that produce libs and final is the executable. I want to enable it only for one of the libs

**Comment by sakra** [comment]
Read the manual.

**Comment by pps83** [comment]
well it doesn't work, because it's intrusive.
I have regular project, some files use my own header and I want to precompile it. In the middle of the entire manual I found [how to use my own header](https://github.com/sakra/cotire/blob/master/MANUAL.md#using-a-manually-maintained-prefix-header-instead-of-the-automatically-generated-one) but now I see it pushes it into every file that didn't even include it originally.
On top of that for whatever random reason it simply doesn't work for me. No clue, make fails after it tries to compile first cpp file from my project and even in VRBOSE=1 there is no error messages. It just doesn't work.

**Comment by pps83** [comment]
When I tried to run manually resulting g++ cmd line I got no error messages, it simply didn't work and didn't produce object file. When I removed -include "....cotire/App_CXX_prefix.hxx" from command line everything works fine. Somehow generated precompiled header breaks it. I also tried to use [cmake-precompiled-header](https://github.com/larsch/cmake-precompiled-header) and I didn't have that problem. What I see however is that precompiled headers generated with cotire for some weird reason are 500MB, maybe that huge size somehow kills gcc silently
